html/client.html

```diff
@@ -16,7 +16,7 @@
 </head>
 
 <body>
-<div id="map" class="map" tabindex="1">
+<div id="mymap" class="map" tabindex="1">
```

Comment:
I don't like the naming with a prefixed "my". Shouldn't we change it to "g4u-map"? The same applies to the configurations.